docs/blocks-storage/querier.md

```diff
@@ -1441,4 +1441,8 @@ blocks_storage:
     # be out-of-order.
     # CLI flag: -blocks-storage.tsdb.out-of-order-cap-max
     [out_of_order_cap_max: <int> | default = 32]
+
+    # [EXPERIMENTAL] True to enable native histogram.
```

Comment:
we should update https://github.com/cortexproject/cortex/blob/master/docs/configuration/v1-guarantees.md too